env.json

```diff
@@ -1,7 +1,7 @@
 {
   "MONGO_HOSTNAME": {
     "type": "string",
-    "default": "mongodb://rems-admin-pims-root:rems-admin-pims-password@localhost:27017",
```

Comment:
We should keep this url the same as it was before as this is what pims uses to connect to mongo.